app/views/import_datasets/_import_dataset.json.jbuilder

```diff
@@ -0,0 +1,19 @@
+# TODO: This view is DwC-A specific placed on a generic directory, it should not.
```

Comment:
Necessary TODO?

Reply:
Could wait after merge and another importer is build on top of ImportDataset?

Reply:
Defnitely